src/esp/geo/VoxelGrid.cpp

```diff
@@ -198,129 +106,143 @@ Mn::Vector3 VoxelGrid::getGlobalCoords(const Mn::Vector3i& coords) {
 }
 
 int VoxelGrid::generateBoolGridFromIntGrid(const std::string& intGridName,
-                                           std::string boolGridName,
+                                           const std::string& boolGridName,
```

Comment:
Could this be implemented using VoxelGrid's public interface? If so, consider making this a free function, to avoid class bloat. https://ict.senecacollege.ca/~oop244/pages/content/nonme.html